dip-template/runtimes/dip-consumer/src/lib.rs

```diff
@@ -119,6 +116,21 @@ pub const HOURS: BlockNumber = MINUTES * 60;
 pub const UNIT: Balance = 1_000_000_000_000;
 pub const MILLIUNIT: Balance = UNIT / 1_000;
 
+/// Maximum number of blocks simultaneously accepted by the Runtime, not yet included into the
+/// relay chain.
+pub const UNINCLUDED_SEGMENT_CAPACITY: u32 = 1;
```

Comment:
Can you move these to just before they are used, and not make them `pub`?

Reply:
reordered in: [9d7ccf2](https://github.com/KILTprotocol/kilt-node/pull/627/commits/9d7ccf29949061d77a273dc6922a3d703e9e17fb)